gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/cors/CorsFilter.java

```diff
@@ -62,33 +62,18 @@ public boolean handleRequest(RequestContext requestContext) {
                             "be responded from the upstream");
                     return true;
                 }
-                StringBuilder allowedMethodsBuilder = new StringBuilder(HttpConstants.OPTIONS);
-                // Handling GraphQL post requests
-                // Only post method is allowed for GQL apis, hence it will be added to the allowed method list.
-                if (APIConstants.ApiType.GRAPHQL.equalsIgnoreCase(requestContext.getMatchedAPI().getApiType())) {
-                    allowedMethodsBuilder.append(", ").append(ResourceConfig.HttpMethods.POST);
-                } else {
-                    for (ResourceConfig resourceConfig : requestContext.getMatchedAPI().getResources()) {
-                        if (!resourceConfig.getPath().equals(requestContext.getRequestPathTemplate())) {
-                            continue;
-                        }
-                        allowedMethodsBuilder.append(", ").append(resourceConfig.getMethod().name());
-                    }
-                }
-                requestContext.getProperties()
-                        .put(APIConstants.MessageFormat.STATUS_CODE, HttpConstants.NO_CONTENT_STATUS_CODE);
-                requestContext.addOrModifyHeaders(HttpConstants.ALLOW_HEADER, allowedMethodsBuilder.toString());
```

Comment:
why did you remove it?

Reply:
Resolved by https://github.com/wso2/apk/pull/1236/commits/d507691ca795a4060ed5fc771dac025b299d23a2